### PR TITLE
Add ADOPTERS file

### PR DESCRIPTION
**Pull request by davidmirror-ops**
Signed-off-by: David Espejo <daespejo@vmware.com>

## Summary

This file will collect the public list of organizations that are using Paketo Buildpacks in production
## Checklist

* [ ] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [ ] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [ ] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by fg-j** [comment]
If we're already aware of adopters, should we add them here? Or request that they add themselves?

**Comment by ForestEckhardt** [comment]
@paketo-buildpacks/steering-committee I would like a little feedback from y'all on this.

File: ADOPTERS.md
Comment (dmikusa-pivotal) [inline ADOPTERS.md:3]: Was there supposed to be a link for `this Discussion`?

File: ADOPTERS.md
Comment (dmikusa-pivotal) [inline ADOPTERS.md:5]: Same. Is this supposed to link to `Adding your organization to the list of Paketo Buildpacks Adopters`? or maybe just say, "see the 'Adding your organization to the list of Paketo Buildpacks Adopters' section below`.

File: ADOPTERS.md
Comment (dmikusa-pivotal) [inline ADOPTERS.md:10]: What is allowed to be included here? Is it just a text list of the company names? Are companies allowed to include a link, perhaps to a blog post or announcement about how they are using it? 

If we are allowing a link, I think we should spell out that the link must point to either the company's home page or an announcement directly mentioning their usage of Paketo buildpacks. Just to prevent folks from trying to link to sales material or other things.

**Comment by dmikusa-pivotal** [comment]
I think we should wait till we have at least one company that has given permission before we merge this. IMHO, it looks incomplete to have this file with no companies listed. We don't want to give the impression no one is using Paketo.

Side note: @davidmirror-ops - Do we need permission if a company has publicly indicated their usage of Paketo buildpacks somewhere, like in their docs?

File: ADOPTERS.md
Comment (davidmirror-ops) [reply ADOPTERS.md]: Yes, it is fixed

File: ADOPTERS.md
Comment (davidmirror-ops) [reply ADOPTERS.md]: Done

**Comment by davidmirror-ops** [comment]
Hey @dmikusa-pivotal yes, even in that case we need to reach out and ask for permission. We can condense a short list of those projects you know are Paketo adopters so I can reach out to them

**Comment by ForestEckhardt** [comment]
@davidmirror-ops bump

**Comment by davidmirror-ops** [comment]
Just added further instructions on how companies/orgs could add themselves to the list

**Review by ForestEckhardt — APPROVED** [review]

**Comment by ForestEckhardt** [comment]
I am happy enough with this. Can I get 1 more @paketo-buildpacks/content-maintainers to approve this?

File: ADOPTERS.md
Comment (fg-j) [inline ADOPTERS.md:5]:
```suggestion
If you want to add it yourself, please follow [these directions](#adding-your-organization-to-the-list-of-paketo-buildpacks-adopters).
```

**Review by fg-j — APPROVED** [review]

**Review by fg-j — APPROVED** [review]

**Pull request merged**
